src/dao/pages/CreateADAO/forms/TokenDAOVotingForm.tsx

```diff
@@ -111,7 +111,7 @@ export function TokenDAOVotingForm() {
             label: "Voting strategy",
             type: "text",
             placeholder: "Enter strategy",
-            unit: "Blocks",
+            unit: "Secs",
```

Comment:
Let's remove this comment. We no longer have plans to address this anytime soon.

Reply:
Done.